### DISCUSSION TITLE
Multiple certificate stores on the same path

### DISCUSSION THREAD

**Discussion by cailloumajor** [locka99/opcua#268] [Q&A]
The documentation of [CertificateStore](https://docs.rs/opcua/0.11.0/opcua/crypto/certificate_store/struct.CertificateStore.html#method.new) states that:

> It is a bad idea to have more than one running instance pointing to the same path location on disk.

What risks do I run for by doing so ?

**Comment by locka99** [comment]
I'd be thinking of potential data races where two processes are reading and writing files to the same directories.
